pkg/storage/inmemory/article.go

```diff
@@ -0,0 +1,97 @@
+package inmemory
+
+import (
+	"errors"
+	"github.com/reiver/go-porterstemmer"
+	"news-aggregator/pkg/model"
+	"strings"
+	"time"
+)
+
+// ArticleInMemory is a struct that contains the in-memory database for articles.
+type ArticleInMemory struct {
+	Articles []model.Article
+	nextID   int
```

Comment:
strage.. how is this used?

Reply:
This id is used specially for InMemory DB implementation, because we need to increment next ID somehow